applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+Utilizing NFTAA provides more methods for managing and delegating staking rights without exposing private keys.
+This approach significantly reduces the risk of unauthorized access and enhances overall security.
+Furthermore, NFTAA allows staked assets to remain liquid, enabling users to trade, lend, or use them as collateral even
+while they are staked.
+This flexibility is crucial for users who need access to their assets without compromising their staking benefits.
+The project also potentially leverages Polkadot's ability to connect multiple blockchains, ensuring seamless asset transfers and
+interactions across different networks.
+This interoperability is essential for creating a cohesive and efficient blockchain ecosystem.
+Moreover, NFTAA empowers users by allowing them to delegate staking rights, transfer ownership of staked assets, and
+interact with multiple blockchain networks through a single NFTAA, thereby enhancing user control and simplifying asset
+management.
+
+The needs addressed by the NFTAA project were identified through a comprehensive approach.
+A literature review of existing research on staking mechanisms, blockchain security, and asset management highlighted
+the limitations and potential improvements in current systems.
+Case studies examining current issues faced by stakers and asset managers in blockchain networks like Ethereum and
+Polkadot provided practical insights into the challenges and opportunities for innovation.
+Additionally, research on the drawbacks of existing staking mechanisms and the potential benefits of using NFTs for
+account management provided a solid foundation for the project's conceptual framework.
+
+There are similar projects in the Polkadot universe:
+
+- Lido: A liquid staking protocol that allows users to stake assets and receive derivatives representing the staked
+  amount.
+- Bifrost: Provides liquidity for staked assets through a voucher system and supports cross-chain staking.
+- Acala: Offers a similar staking mechanism with its LDOT token, providing immediate liquidity and flexibility in
+  staking.
+
+Differences and Distinctions:
+
+- NFTAA vs. Lido/Bifrost/Acala: Unlike these projects, NFTAA focuses on using NFTs as proxy accounts for staking, which
+  allows for a more flexible and secure delegation of staking rights and ownership. NFTAA also leverages the composability
+  of NFTs, enabling them to hold other assets and act as accounts within the blockchain ecosystem.
+
+Similar concepts exist in other ecosystems:
+
+- Rocket Pool: An Ethereum-based protocol for decentralized staking that allows users to earn rewards without running a
+  full node.
+- EIP-6551 (TBA - token bound accounts): A proposed Ethereum standard to enhance NFTs by equipping them with smart contract capabilities, allowing them to own assets and interact with other contracts.
+
+Differences and Distinctions (NFTAA vs. EIP-6551):
```

Comment:
Why is an EIP a key reference here, this is Substrate after all and not an EVM chain which we are talking about. Without defining how you translate the EIP to a Substrate equivalent this is hard to follow.

Reply:
This is just related work and mainly a comparison of business characteristics not a comparison of technicalities. We will not translate EIP to substrate equivalent.